install/view/template/settings.tpl

```diff
@@ -35,17 +35,15 @@
                 <input type="password" name="admin_password" value="<?php echo $admin_password; ?>" placeholder="<?php echo $entry_admin_password; ?>" id="input-admin-password" class="form-control" />
             </div>
         </div>
-
         <div class="form-group pull-left">
             <div class="col-xs-12">
-                <button type="button" onclick="displayDatabase();" class="btn btn-primary"><i class="fa fa-arrow-left"></i> <?php echo $button_back; ?></button>
+                <button type="button" onclick="displayDatabase();" class="btn btn-info"><i class="fa fa-arrow-left"></i> <?php echo $button_back; ?></button>
```

Comment:
I would prefer this as btn-default. With btn-info we another ton of blue color to the screen.
![2015-06-06_2100](https://cloud.githubusercontent.com/assets/5254835/8020790/1ffe627a-0c8f-11e5-952d-c68f750bdfb8.png)